src/common/components/addressForm/addressForm.tpl.html

```diff
@@ -134,10 +135,51 @@
           <div role="alert" ng-messages="$ctrl.parentForm.addressRegion.$error" ng-if="($ctrl.parentForm.addressRegion | showErrors)">
             <div class="help-block" ng-message="required" translate>{{'SELECT_STATE_ERROR'}}</div>
           </div>
+        </div>
+      </div>
+      <div class="col-sm-6 col-md-3">
```

Comment:
It would be really nice to implement this without duplicating the state and ZIP code fields. Maybe I'm missing something, but it seems that just changing the `col-xx-n` class would be enough without the `ng-if`s.